### DISCUSSION TITLE
JitPack artifacts

### DISCUSSION THREAD

**Discussion by matytyma** [bailuk/java-gtk#25] [Q&A]
Are there still artifacts published to JitPack? I've tried
```kts
repositories {
    maven { url = uri("https://jitpack.io") }
}

dependencies {
    implementation("com.github.bailuk:java-gtk:0.4.0")
}
```
but could not get the artifact (as it's not a part of the GH release? :thinking:)

**Comment by bailuk** [comment]
You need to add mavenCentral

```kotlin
repositories {
    mavenCentral()
    maven { url = uri("https://jitpack.io") }
}
```
You can test accessibility of artifacts here:
Info: https://jitpack.io/#bailuk/java-gtk/0.4.0
Artifacts: https://jitpack.io/com/github/bailuk/java-gtk/v0.4.0

Depending on the project structure you need to replace 'implementation' with 'api' to export symbols.

**Reply by matytyma** [reply]
> You need to add mavenCentral

I've already had Maven central in my build file, I have just ommited that in what I've sent :smile:

> Depending on the project structure you need to replace 'implementation' with 'api' to export symbols.

The change from `implementation` to `api` only changes the output after reloading Gradle... :sweat_smile:
